transference-service/src/main/java/com/insight/learning/platabank/transferenceservice/service/impl/TEDTransferenceServiceImpl.java

```diff
@@ -1,15 +1,17 @@
 package com.insight.learning.platabank.transferenceservice.service.impl;
 
-import com.insight.learning.platabank.transferenceservice.dto.TransferenceDTO;
+import com.insight.learning.platabank.transferenceservice.domain.Transference;
 import com.insight.learning.platabank.transferenceservice.service.TransferenceService;
 import org.springframework.stereotype.Service;
 
+import java.util.Optional;
+
 @Service
 public class TEDTransferenceServiceImpl extends TransferenceService {
 
     @Override
-    public TransferenceDTO doTransference(TransferenceDTO transferenceDTO) {
-        return null;
+    public Optional<Transference> doTransference(Transference transference) {
+        return Optional.empty();
```

Comment:
?
Se ainda for implementar a lógica, nesses casos é legal por um //TODO: xxxxxx